examples/with-opentelemetry/instrumentation.ts

```diff
@@ -0,0 +1,32 @@
+export function register() {
+  if (process.env.NEXT_RUNTIME === 'nodejs') {
+    // We need to make sure that we import these files only in Node.js environment.
+    // OpenTelemetry is **not** supported on Edge or Client side at the moment.
+    const { Resource } = require('@opentelemetry/resources')
+    const {
+      SemanticResourceAttributes,
+    } = require('@opentelemetry/semantic-conventions')
```

Comment:
an easier construct would be to import a `instrumentation-node` in the if statement, that way we don't need to write requires manually and can use the regular syntax

Reply:
I used that when dogfooding sync-linear.

I was afraid that that would make the distinction between node and edge less explicit, but that is probably fine.